.ci/run_examples.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
Make sure both of the following are used in both scripts.
```
set -e
trap exit SIGINT
```

The first will make the script exit if any command in it exits with a non-zero status.
The second will pass along Ctrl-C's to the subcommand instead of ignoring them.